src/clr-angular/utils/_helpers.clarity.scss

```diff
@@ -13,15 +13,6 @@
   @return $remSize * $clr-font-base-size * 1px;
 }
 
-//Get a selector for all the input types specified
-@function inputs($types...) {
-  $inputs: ();
-  @each $type in $types {
-    $inputs: $inputs, unquote('input[type=' + $type + ']');
-  }
-  @return quote($inputs);
```

Comment:
Dart Sass implementation of quote did not work with maps. I removed the `inputs` function altogether as it was only being used in the Stack View component.